src/styles/exchange.css

```diff
@@ -0,0 +1,190 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Pacifico&family=Poppins:wght@700&display=swap');
+
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #578ae5;
+}
+
+.header {
+  padding: 1rem;
+  background-color: #4269b2;
+}
```

Comment:
- [ ] As it is a requirement that your app matches the given wireframe, kindly add the section highlighted in the image below to your design

![Screenshot 2023-06-11 at 00 23 02 (2)](https://github.com/abiola-arobieke/fx-today/assets/64774936/43355c5f-73bd-4294-9dfe-abce9ab8d243)